src/graphql/root/mutation/user-contact-update-alias.ts

```diff
@@ -27,19 +27,19 @@ const USerContactUpdateAliasMutation = GT.Field({
       }
     }
 
-    const user = await Users.updateContactAlias({
+    const contact = await Users.updateContactAlias({
       userId: uid as UserId,
       username,
       alias,
     })
 
-    if (user instanceof Error) {
-      return { errors: [{ message: user.message }] }
+    if (contact instanceof Error) {
+      return { errors: [{ message: contact.message }] }
     }
 
     return {
       errors: [],
-      user,
+      contact,
```

Comment:
`SuccessPayload` type has `success` key here instead I believe

Reply:
Rebased to fix this and merge with new description fields.

I made the payload here to be a contact user.